docs/operator-docs/kinesis-stream-for-ingest.md

```diff
@@ -0,0 +1,26 @@
+---
+id: kinesis-stream-for-ingest
+title: Kinesis Stream For Ingest
+hide_title: true
+---
+
+# Setup Kinesis Stream & CNM Message
+
+> **Note**: Keep in mind that you should only have to set this up once. This should be created via the SDS or from a cloud formation template. Kinesis pricing is based on the shard value and not on amount of kinesis usage.
+
+1. Create a Kinesis Stream
+
+* Go to the `Kinesis` service and click `Create Data Stream`.
+* Assign a name to the stream.
+* Apply a `shard value` of `1`.
+* Click on `Create Kinesis Stream`.
+* A status page with stream details display. Once the status is `active` then the stream is ready to use. Keep in mind to record the streamName and StreamARN for later use.
+
+2. Create a Rule
+
+* Refer to [Create Rule in Cumulus](../operator-docs/create-rule-in-cumulus).
+
+3. Send a CNM Message
```

Comment:
CNM is kind of an EED thing. It doesn't have to be a CNM message it just has to be a message matching the schema: https://github.com/nasa/cumulus/blob/master/packages/api/lambdas/kinesis-consumer-event-schema.json

Reply:
@laurenfrederick Changed from CNM message to message.